cibuild.cmd

```diff
@@ -1,6 +1,14 @@
 call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
 msbuild /v:m /m BuildAndTest.proj /p:CIBuild=true
+if ERRORLEVEL 1 (
+    echo Build failed
+    exit /b 1
+)
 
 REM Kill any instances of VBCSCompiler.exe to release locked files;
 REM otherwise future CI runs may fail while trying to delete those files.
-taskkill /F /IM vbcscompiler.exe
\ No newline at end of file
+taskkill /F /IM vbcscompiler.exe
```

Comment:
This is yet another reason why VBCSCompiler needs to reconsider the idea of a timeout.  Life would be so much easier here if it were just tied to the lifetime of MSBuild.